lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+import { Subject } from 'rxjs';
+import { SearchDateRangeAdvanced } from './search-date-range-advanced/search-date-range-advanced';
+import { DateRangeType } from './search-date-range-advanced/date-range-type';
+import { SearchWidget } from '../../models/search-widget.interface';
+import { SearchWidgetSettings } from '../../models/search-widget-settings.interface';
+import { SearchQueryBuilderService } from '../../services/search-query-builder.service';
+import { InLastDateType } from './search-date-range-advanced/in-last-date-type';
+
+@Component({
+  selector: 'adf-search-date-range-advanced-tabbed',
+  templateUrl: './search-date-range-advanced-tabbed.component.html'
+})
+export class SearchDateRangeAdvancedTabbedComponent implements SearchWidget {
+    displayValue$ = new Subject<string>();
+    id: string;
+    startValue: SearchDateRangeAdvanced = {
+        dateRangeType: DateRangeType.ANY,
+        inLastValueType: InLastDateType.DAYS,
+        inLastValue: undefined,
```

Comment:
Why do you need to set these 3 fields to undefined? You can skip inLastValue, betweenStartDate and betweenEndDate from that object because they are marked as optional in SearchDateRangeAdvanced so if you skip them they will be treated like undefined

Reply:
Skipping on the 3 values causes issues with the clear button. It resets the radio button selection to Anytime, but does not clear out any values that may have been entered into the form fields